extensions/wikia/Chat2/js/ChatEntryPoint.js

```diff
@@ -93,6 +93,13 @@ var ChatEntryPoint = {
 			var $e = $(this);
 			$e.html($.msg($e.data('msg-id'), $e.data('msg-param')));
 		});
+
+        //enables live-chat styling only on one page -
+        //chat couldn't be tested on devbox/sandbox
+        // so we'll see how it behaves on production
+        if (window.wgTitle == 'Chat test diana') {
```

Comment:
I would vote for not merging this to dev.
For test purposes - ok. But still we can achieve the same result  by adding class to WikiaRail in browser.
With this thing you need to remember to remove this after it is tested.